editor.planx.uk/src/@planx/components/Calculate/Editor.tsx

```diff
@@ -123,10 +120,10 @@ export default function Component(props: Props) {
           {variables?.length ? (
             <>
               <p>If</p>
-              {variables.map((variable, i) => (
+              {variables.map((variable) => (
                 <InputRow key={variable}>
                   <Input value={variable} disabled />
-                  <span className={classes.were}>were</span>
+                  <Were>were</Were>
```

Comment:
nit: totally appreciate this isn't introduced here, but this language feels a little awkward ! 

in the future, perhaps we can simplify to "If {x} **'is** {y}, then {fn} **will be** set to {output}" - but happy for this to be revisited, no need to open that can of worms now

Reply:
Yeah this is totally fair - I felt this was awkward and couldn't hit on a good name. Will go for `ConditionLabel` thanks 👍